### DISCUSSION TITLE
Can I get back the wrong fee?

### DISCUSSION THREAD

**Discussion by shhuyi** [Chia-Network/chia-blockchain#6806] [Support]
![20210616203446](https://user-images.githubusercontent.com/12879302/122219847-6b1e2300-cee2-11eb-8c4b-352a2656ca6d.png)

**Comment by loppefaaret** [comment]
no, it is gone to who ever won that block that contained the transaction unfortunately

**Comment by BrandtH22** [comment]
@shhuyi , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
